llvm/test/Transforms/LoopVectorize/AArch64/clamped-trip-count.ll

```diff
@@ -0,0 +1,32 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 2
```

Comment:
nit: This comment isn't true anymore

Reply:
Ah, my bad for not removing it, I missed it.